README.md

```diff
@@ -225,7 +232,7 @@ Indentation types are:
 * `:inner` -
   two character indentation applied to form arguments at a depth
   relative to a form symbol
-  
```

Comment:
Remove any formatting corrections not relevant to the PR. It makes the diff much easier to read.

Reply:
done